test/functorch/test_ops.py

```diff
@@ -1884,6 +1895,126 @@ def f(x):
                 with self.assertRaisesRegex(RuntimeError, "During a grad .* attempted to call in-place operation"):
                     vjp(f, torch.randn_like(without_grad))
 
+    @_set_autograd_function_extension_enabled()
```

Comment:
We should probably also add at least one test that does a composition with jvp since all the autograd function tests have these defined

Reply:
Yeah, to test `generate_vmap_rule` for jvp, we need at least the following tests (that don't exist yet)
- jvp x vmap
- jvp x vmap x vmap
- possibly more... jvp x jvp x vmap, vmap x jvp x vmap depending on how unconvinced we are

Was planning to punt that to a follow-up PR before I add jvp support for generate_vmap_rule, unless you want to see all of those in this PR

Reply:
follow up PR works for me